apps/web/src/app/(private)/projects/[projectId]/versions/[commitUuid]/documents/_components/BreadcrumpInput.tsx

```diff
@@ -21,7 +24,9 @@ export default function BreadcrumpInput({ projectId }: { projectId: number }) {
     { trailing: true },
   )
 
-  if (!project) return null
+  if (!project) {
+    return <Text.H5M color='foregroundMuted'>{projectName}</Text.H5M>
+  }
 
   return (
     <EditableText
```

Comment:
now that you are at it fix the typo in the file **Breadcrumb**

Reply:
omg I did 1k times this typo 🙈 😂 🙈